pycheribuild/projects/disk_image.py

```diff
@@ -377,7 +392,6 @@ def path_relative_to_outputroot(xtarget) -> Path:
         if not sshd_config.exists():
             self.info("SSHD not installed, not changing sshd_config")
         else:
-            self.info("Adding 'PermitRootLogin without-password\nUseDNS no' to /etc/ssh/sshd_config")
```

Comment:
Where'd this go?

Reply:
Seems to have been a rebase error after 8af1669bf9b4e4daaae1ee612550a7e9fe895a3a landed. Weird.